src/main/java/net/gegy1000/plasmid/shop/Cost.java

```diff
@@ -36,6 +36,18 @@ public static Cost ofEmeralds(int emeralds) {
                 .display(new LiteralText(emeralds + " emeralds").formatted(Formatting.GREEN));
     }
 
+    public static Cost free(){
+        return new Cost()
+                .take(Items.AIR, 0)
```

Comment:
Maybe pass the `take` function and just return true

Reply:
what do you mean by that?

Reply:
There's an overload to `take` which takes a function. Instead of trying to take an empty stack from the inventory, it could no-op directly and return successful (true)

Reply:
ah i see. sorry didnt see that